README.md

```diff
@@ -30,7 +30,22 @@ GITHUB_URL=<your_github_url where the authorization code lives>
 - Run the application from the entry point
 
 ```console
-$ export PYTHONPATH=$PWD && python app/main.py
+$ python3 main.py
+```
+
+- Run the application from the docker image
+```console
+$ docker run rscheele3214/rego_builder
+```
```

Comment:
It should be a generic name like `geobeyond/policy-builder-api`. There's no instruction as to how to build the image as well - add a section on building the image from scratch.

Reply:
We'd have to move this to the company's account. This is just a demo

Reply:
> It should be a generic name like `geobeyond/policy-builder-api`. There's no instruction as to how to build the image as well - add a section on building the image from scratch.

I'll add the instructions to the README